app/check.php

```diff
@@ -21,7 +21,7 @@
 
 $messages = array();
 foreach ($symfonyRequirements->getRequirements() as $req) {
-    /** @var $req Requirement */
+    /* @var $req Requirement */
```

Comment:
This regression has been fixed in php-cs-fixer now. It no longer makes this false change.